src/verify/checks/_py/mutate.py

```
@@ -32,37 +42,165 @@
+        if (
+            isinstance(first, ast.Expr)
+            and isinstance(first.value, ast.Constant)
+            and isinstance(first.value.value, (str, bytes))
+        ):
+            c = first.value
+            spans.append((c.lineno, c.col_offset, c.end_lineno, c.end_col_offset))
+    return spans
+
+
+def _in_docstring(spans: list, row: int, col: int) -> bool:
+    for (l, c, el, ec) in spans:
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)